shared/django_apps/timeseries/migrations/0016_auto_20250206_1513.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.16 on 2025-02-06 15:13
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # cant create this views in a transaction
+    atomic = False
+
+    dependencies = [
+        ("timeseries", "0015_testrun_testrun_flags_hash_test_id_unique"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            """
+                create materialized view timeseries_testrun_branch_summary_1day
+                with (timescaledb.continuous) as
+                select
+                    repo_id,
+                    branch,
+                    testsuite,
+                    classname,
+                    name,
+                    min(computed_name) as computed_name,
+                    COUNT(DISTINCT CASE WHEN outcome = 'failure' OR outcome = 'flaky_fail' THEN commit_sha ELSE NULL END) AS cwf,
+                    time_bucket(interval '1 days', timestamp) as timestamp_bin,
```

Comment:
```suggestion
                    time_bucket(interval '1 days', timestamp) as timestamp_bin,
                    
                    COUNT(DISTINCT CASE WHEN outcome = 'failure' OR outcome = 'flaky_fail' THEN commit_sha ELSE NULL END) AS failing_commits,
```

please spell out `failing_commits` instead of the cryptic `cwf`.

also IMO its a bit more readable to visually group / separate the "group by" columns and the aggregates.